pglite/cibuild.sh

```diff
@@ -240,15 +240,6 @@ END
 
     . ${PGROOT}/pgopts.sh
 
-    # make sure no non-mvp feature gets in.
```

Comment:
some compilers will try to use cpu intrinsics by default and some of them are allowed or emulated but outside MVP which may breaks compat on mobile browsers. Better force off to keep close to wasm MVP